src/components/EmailItem/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+function EmailItem({content, address, name, subject, body, date, id, myFunction, read}) {
+
+    let cs = 'bg-gray-100 p-[10px] border-2 border-black hover:bg-blue-500 text-left';
+    let isRead = 'UNREAD';
+
+    if (read == 1) {
+        cs = 'bg-gray-500 p-[10px] border-2 border-black hover:bg-blue-500 text-white text-left';
+        isRead = 'READ';
+    }
+
+    return (
+        <button id={id} onClick={myFunction} data-id={id} data-content={content} data-address={address} data-name={name} data-subject={subject} data-body={body} data-date={date} data-read={read}>
+            <div className={cs}>
+                <div className='flex justify-between'>
+                    <div className='flex flex-col'>
+                        <p>{name}</p>
+                        <p>{subject}</p>
+                        <p>{body}</p>
+                    </div>
+                    <div className='flex flex-col justify-between text-right'>
+                        <p>{date}</p>
```

Comment:
Somewhere in the last merge we lost the proper date formatting that was done here. Please reimplement this.

Reply:
This last merge also lost styling.